File: artifacts/mimalloc/decompose/src/mi_visit_blocks_args_t.c
typedef struct mi_visit_blocks_args_s
{
  bool visit_blocks;
  mi_block_visit_fun *visitor;
  void *arg;
} mi_visit_blocks_args_t
/*** DEPENDENCIES:
typedef bool mi_block_visit_fun(const mi_heap_t *heap, const mi_heap_area_t *area, void *block, size_t block_size, void *arg)
----------------------------
***/

File: artifacts/mimalloc/manual_debug/src/mi_visit_blocks_args_t.rs
use crate::*;
use std::sync::atomic::AtomicUsize;
use std::sync::atomic::Ordering;


#[derive(Clone)]
pub struct MiVisitBlocksArgsS {
    pub visit_blocks: bool,
    pub visitor: Option<MiBlockVisitFun>,
    pub arg: Option<Vec<u8>>,
}

pub type MiVisitBlocksArgsT = MiVisitBlocksArgsS;

